infra/src/test/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSwitchServiceIT.kt

```diff
@@ -616,6 +616,65 @@ constructor(
         )
     }
 
+    @Test
+    fun `findSwitchesRelatedToOperationalPoint ignores NOT_EXISTING switches`() {
+        val operationalPoint =
+            mainDraftContext
+                .save(
+                    operationalPoint(
+                        polygon =
+                            Polygon(
+                                Point(0.0, 0.0),
+                                Point(10.0, 0.0),
+                                Point(10.0, 10.0),
+                                Point(0.0, 10.0),
+                                Point(0.0, 0.0),
+                            )
+                    )
+                )
+                .id
+        val inAreaExisting =
+            mainDraftContext
```

Comment:
Silloin kun formatointi sisentää näin ärsyttävästi niin olen yleensä vain luonut tallennettavan asian (tässä switch tai vain joint-list) erikseen muuttujaan yllä. Toki voi myös tehdä let-rakenteena:
```
switch(joints = listOf(LayoutSwitchJoint(JointNumber(1), SwitchJointRole.MAIN, Point(10.0, 10.0), null)))
  .let(mainDraftContext::save)
  .id
```